### ISSUE TITLE
Golem w/ USE seal cannot place sapling.

### ISSUE THREAD

**Issue by VadersFist0501** [Azanor/thaumcraft-beta#1427]
In Beta23 I had an automagic treefarm set up using Golems and USE seals. After the upgrade to Beta24, the Golems could no longer place saplings down. They would attempt to place the sapling (placing animation) indefinitely, or until they returned the sapling to their chest. No saplings have been placed since.

**Comment by Omnimetatron** [comment]
https://imgur.com/a/GIFqZTM

I've tried every combination of settings with the provide and use seals to no (well, very little) avail. A couple of times the golem did actually place a sapling only to never do it again. Shift-clicking on the back face of the seal to request the sapling even worked, but just once. Very odd behaviour.

**Comment by Omnimetatron** [comment]
A new development occurred after I picked up and slightly moved a Golem!
https://imgur.com/gallery/cQ9jCQh

**Comment by Azanor** [comment]
If you place the seal on the ground the golem will repeatedly try and use saplings on that spot - the only thing the seal does is tell a golem to click on the marked blocked with an item. To have the golem only replant the sapling when there isn't one there, first place a sapling, then place the seal on the sapling and set the seal to only click on an empty space.

**Comment by Omnimetatron** [comment]
Well, now I've done what you said @Azanor I've yet to have a single issue, aside from a single Oak Sapling that lost its texture but was placed in the white-listed sapling container anyway. Still no idea what was causing that, aside from the fact that I did originally have the Use Seals placed on the side of a block that I then removed. All my Use Seals are now placed on top of the sapling and I'm not seeing that issue anymore.

**Comment by Azanor** [comment]
When the sapling with the broken texture was placed in the container did it retain the broken texture, or did it stack with the normal saplings?

**Comment by VadersFist0501** [comment]
The USE seals are floating one block in the air.
The setup I am using is the same one used in TC5 and TC6-Beta23
The only difference is that the golem has stopped placing saplings, when they did perfectly a version before.

**Comment by Omnimetatron** [comment]
@VadersFist0501 Have you tried picking up the golem and placing it down in the world again?
https://imgur.com/ckqMo0n
I have had no issues since I followed @Azanor's instructions, though I did get a single sapling with a missing texture, I think this was because the golems were trying to provide a sapling to a spot while I was setting up the Use Seals. I've had my tree farm running for hours now, and it's working perfectly.
